python/hsfs/feature_group.py

```diff
@@ -116,8 +116,35 @@ def select_all(self, include_primary_key=True, include_event_time=True):
 
             # show first 5 rows
             query.show(5)
+
+
+            # select all features exclude primary key and event time
+            from hsfs.feature import Feature
+            fg = fs.create_feature_group(
+                    "fg",
+                    features=[
+                            Feature("id", type="string"),
+                            Feature("ts", type="bigint"),
+                            Feature("f1", type="date"),
+                            Feature("f2", type="double")
+                            ],
+                    primary_key=["id"],
+                    event_time="ts")
+
+            query = fg.select_all()
+            query.features
+            # [Feature('id', ...), Feature('ts', ...), Feature('f1', ...), Feature('f2', ...)]
+
+            query = fg.select_all(include_primary_key=False, include_event_time=False)
+            query.features
+            # [Feature('f1', ...), Feature('f2', ...)]
             ```
 
+        # Arguments
+            include_primary_key: boolean, optional. If True, include primary key of the feature group
+            to the feature list. Defaults to True.
+            include_event_time: boolean, optional. If True, include event time of the feature group
+            to the feature list. Defaults to True.
```

Comment:
In general you shouldn't need to describe the types manually if you put the type annotations in the method signature. For example, see here:
![image](https://user-images.githubusercontent.com/8422705/210340037-1f56c2e5-13d1-4115-bc78-4f261b4069f6.png)


Should be indented.
```suggestion
            include_primary_key: boolean, optional. If True, include primary key of the feature group
                to the feature list. Defaults to True.
            include_event_time: boolean, optional. If True, include event time of the feature group
                to the feature list. Defaults to True.
```